HSLdevcom/hsl-map-web-ui
# WIP: MM-283: Disruption alerts for map-web-ui

Also check: https://github.com/HSLdevcom/transitlog-alert-proxy/pull/1

How to test: 
- Spin up the transitlog-alert-proxy
- start map-web-ui locally
- Choose lines, then navigate to disruptions tab.

Still needs to check environment configurations.
